java/src/test/java/ai/rapids/cudf/ReductionTest.java

```diff
@@ -612,7 +612,7 @@ void testWithSetOutputType() {
       assertEquals(expected, result);
     }
 
-    try (Scalar expected = Scalar.fromFloat(1.666667f);
+    try (Scalar expected = Scalar.fromFloat(1.6666666f);
```

Comment:
This is the closest float to the exact value of $\frac{5}{3}$, which we now obtain.